### PR TITLE
Rel-984674 import service staging governance correct loadfile paths in relativity import samples

### PR DESCRIPTION
**Pull request by annasojkapal**
Changed sample paths to paths located in \StructuredData\Import\ in:
- openapidoc
- readme
- dotNet samples
- Kepler samples
- REST samples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1187,7 +1187,7 @@ List of samples:
 - Update variable which defines the path to the load file/opticon file (e.g. $opticonFilePath) according to the location where you copied sample data.
 
         $workspaceId = 1000000
-        $loadFilePath = "\\files\T001\Files\SampleDataSources\load_file_01.dat"
+        $loadFilePath = "\\\\files\T001\StructuredData\\Import\SampleDataSources\load_file_01.dat"
```
Comment (dominik-tomasiewicz-rel) [inline README.md:1190]:
```suggestion
        $loadFilePath = "\\\\files\\T001\\StructuredData\\Import\\SampleDataSources\\load_file_01.dat"
```

File: Samples/RestSamples/SamplesCollection/sample01-import-native-files.ps1 [hunk 2]
```diff
@@ -3,7 +3,7 @@
 . "$global:rootDir\Helpers\WriteInformationClass.ps1"
 
 $workspaceId = 1000000
-$loadFilePath = "\\files\T001\Files\SampleDataSources\load_file_01.dat"
+$loadFilePath = "\\\\files\T001\StructuredData\Import\SampleDataSources\load_file_01.dat"
```
Comment (dominik-tomasiewicz-rel) [inline Samples/RestSamples/SamplesCollection/sample01-import-native-files.ps1:6]:
```suggestion
$loadFilePath = "\\\\files\\T001\\StructuredData\\Import\\SampleDataSources\\load_file_01.dat"

```

File: Samples/RestSamples/SamplesCollection/sample02-import-documents-in-overlay-mode.ps1 [hunk 3]
```diff
@@ -3,7 +3,7 @@
 . "$global:rootDir\Helpers\WriteInformationClass.ps1"
 
 $workspaceId = 1000000
-$loadFilePath = "\\files\T001\Files\SampleDataSources\load_file_01.dat"
+$loadFilePath = "\\\\files\T001\StructuredData\Import\SampleDataSources\load_file_01.dat"
```
Comment (dominik-tomasiewicz-rel) [inline Samples/RestSamples/SamplesCollection/sample02-import-documents-in-overlay-mode.ps1:6]:
```suggestion
$loadFilePath = "\\\\files\\T001\\StructuredData\\Import\\SampleDataSources\\load_file_01.dat"

```

File: Samples/RestSamples/SamplesCollection/sample03-import-from-two-data-sources.ps1 [hunk 4]
```diff
@@ -3,8 +3,8 @@
 . "$global:rootDir\Helpers\WriteInformationClass.ps1"
 
 $workspaceId = 1000000
-$loadFile01Path = "\\files\T001\Files\SampleDataSources\load_file_01.dat"
-$loadFile02Path = "\\files\T001\Files\SampleDataSources\load_file_02.dat"
+$loadFile01Path = "\\\\files\T001\StructuredData\Import\SampleDataSources\load_file_01.dat"
```
Comment (dominik-tomasiewicz-rel) [inline Samples/RestSamples/SamplesCollection/sample03-import-from-two-data-sources.ps1:6]:
```suggestion
$loadFile01Path = "\\\\files\\T001\\StructuredData\\Import\\SampleDataSources\\load_file_01.dat"

```

Comment (dominik-tomasiewicz-rel) [reply Samples/RestSamples/SamplesCollection/sample03-import-from-two-data-sources.ps1:6]: and for a lot lines in next samples

**Review by dominik-tomasiewicz-rel — CHANGES_REQUESTED** [review]

File: OpenAPI/openapidoc.json [hunk 5]
```diff
@@ -1473,7 +1473,7 @@
           "Path": {
             "type": "string",
             "description": "File localization",
-            "example": "\\files\\T001\\Files\\tmp_loadfile\\loadfile.dat"
```
Comment (dominik-tomasiewicz-rel) [inline OpenAPI/openapidoc.json:1476]: this openapidoc.json is the same as in our import repo?

if it is then we need to remeber to upadate this after updates in bitbucket

File: README.md (hunk 1 above)
Comment (annasojkapal) [reply README.md]: For rest samples there should be only one slash - I didn't notice incorrect, additional slash - thanks for noticing.

File: Samples/RestSamples/SamplesCollection/sample02-import-documents-in-overlay-mode.ps1 (hunk 3 above)
Comment (annasojkapal) [reply Samples/RestSamples/SamplesCollection/sample02-import-documents-in-overlay-mode.ps1]: Same as above.

File: Samples/RestSamples/SamplesCollection/sample01-import-native-files.ps1 (hunk 2 above)
Comment (annasojkapal) [reply Samples/RestSamples/SamplesCollection/sample01-import-native-files.ps1]: Same as above.

File: Samples/RestSamples/SamplesCollection/sample03-import-from-two-data-sources.ps1 (hunk 4 above)
Comment (annasojkapal) [reply Samples/RestSamples/SamplesCollection/sample03-import-from-two-data-sources.ps1]: Same as above.

File: OpenAPI/openapidoc.json (hunk 5 above)
Comment (annasojkapal) [reply OpenAPI/openapidoc.json:1476]: They are a little bit different, but should probably be the same. Yes - I'll update this file after Igor's changes to original repo.

**Review by dominik-tomasiewicz-rel — APPROVED** [review]

**Review by mateuszKulikRelativity — APPROVED** [review]

**Pull request merged**